targets/Program.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Bullseye;
+using static Bullseye.Targets;
+using static SimpleExec.Command;
+
+internal class Program
+{
+    public static Task Main(string[] args)
+    {
+        var testProjectDirectories = Directory.GetDirectories("tests");
+        var mainProjectFile = "src/SelfInitializingFakes/SelfInitializingFakes.csproj";
+        var releaseNotesFile = "./release_notes.md";
+        var solutionFile = "./SelfInitializingFakes.sln";
+        var versionInfoFile = "./src/VersionInfo.cs";
+
+        var logsDirectory = "./artifacts/logs";
+        var outputDirectory = Path.GetFullPath("./artifacts/output");
+        var testsDirectory = "./artifacts/tests";
+
+        string version = null;
+
+        Targets.Add("default", DependsOn("pack", "test"));

```

Comment:
Now that you're operating within a `Targets` project, I'd get rid of the `Targets` static type qualifier here (you already don't have it on `RunAsync(args)`), but I guess it's down to personal taste.

Reply:
Right. I… think I like the `Targets` for now. Good point about the `RunAsync`!